README.md

```diff
@@ -41,6 +41,17 @@ steps:
     package-name: ament_copyright
 ```
 
+### Build with a custom `repos` or `rosinstall` file
+
+```yaml
+steps:
+- uses: actions/checkout@v2  # checkout rosinstall file
+- uses: ros-tooling/setup-ros@master
+- uses: ros-tooling/action-ros-ci@master
```

Comment:
Could we change this to be a specific version? (Whatever it is currently).
This way we follow the convention [suggested by GitHub](https://github.com/actions/toolkit/blob/master/docs/action-versioning.md)

Reply:
Done.